source/sdk/math.ooc

```diff
@@ -59,6 +59,10 @@ floor: extern(floorl) func ~Long (LDouble) -> LDouble
 	 maximumValue ::= static SHRT_MAX
 }
 
+extend Int64 {
+	modulo: func(divisor: Int) -> This { this - divisor * (this / divisor) }
+}
```

Comment:
1. This implementation differs from the one for `Int`. That means at least one of them is wrong. Perhaps now would be a good time to decide which one is correct?
2. Shouldn't `divisor` also be an `Int64`, i.e. `divisor: This`?